curriculum/challenges/english/25-front-end-development/quiz-javascript-audio-and-video/66edd3b3096349f06cf688bb.md

```diff
@@ -17,439 +17,449 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --distractors--
 
-Placeholder distractor 1
+Increases the volume of the media
 
 ---
 
-Placeholder distractor 2
+Changes the file format of the media
 
 ---
 
-Placeholder distractor 3
+Encrypts the media for secure transmission
 
 #### --answer--
 
-Placeholder answer
+Compresses and decompresses digital media
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which API provides a way to create, process, and analyze audio in web applications?
 
 #### --distractors--
 
-Placeholder distractor 1
+Audio Processing API
 
 ---
 
-Placeholder distractor 2
+Sound Manipulation API
 
 ---
 
-Placeholder distractor 3
+Media Stream API
 
 #### --answer--
 
-Placeholder answer
+Web Audio API
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the purpose of the `getUserMedia()` method?
 
 #### --distractors--
 
-Placeholder distractor 1
+Authenticate users with a media service
 
 ---
 
-Placeholder distractor 2
+Retrieve user preferences for media playback
 
 ---
 
-Placeholder distractor 3
+Download media files to the user's device
 
 #### --answer--
 
-Placeholder answer
+Access the user's camera and/or microphone
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which property of the `HTMLMediaElement` interface represents the current playback time?
 
 #### --distractors--
 
-Placeholder distractor 1
+`time`
 
 ---
 
-Placeholder distractor 2
+`playbackPosition`
 
 ---
 
-Placeholder distractor 3
+`currentPosition`
 
 #### --answer--
 
-Placeholder answer
+`currentTime`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the following code snippet do?
+
+```javascript
+const audio = new Audio('song.mp3');
+audio.loop = true;
+```
 
 #### --distractors--
 
-Placeholder distractor 1
+Plays the audio file once
 
 ---
 
-Placeholder distractor 2
+Increases the volume of the audio
 
 ---
 
-Placeholder distractor 3
+Pauses the audio after one loop
 
 #### --answer--
 
-Placeholder answer
+Sets the audio to repeat indefinitely
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which method is used to capture a media stream from a canvas element?
 
 #### --distractors--
 
-Placeholder distractor 1
+`canvas.getStream()`
 
 ---
 
-Placeholder distractor 2
+`canvas.captureMedia()`
 
 ---
 
-Placeholder distractor 3
+`canvas.toDataURL()`
 
 #### --answer--
 
-Placeholder answer
+`canvas.captureStream()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the purpose of the `MediaRecorder` API?
 
 #### --distractors--
 
-Placeholder distractor 1
+To play audio and video files
 
 ---
 
-Placeholder distractor 2
+To edit existing media files
 
 ---
 
-Placeholder distractor 3
+To create visual representations of audio
 
 #### --answer--
 
-Placeholder answer
+To record audio and video from a media stream
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is NOT a valid state for an `HTMLMediaElement`?
```

Comment:
How exactly do you check the state on an HTMLMediaElement? I'm having trouble finding what you mean. 

https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/paused

https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/playing

https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/ended